requirements.txt

```diff
@@ -1,2 +1,3 @@
 beautifulsoup4>=4.9
 requests>=2.20
+pandas
```

Comment:
Probably add a version just to make sure it works in the future and we have less version issues